microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/jaxrs/JaxRsContext.java

```diff
@@ -75,19 +95,48 @@ public String getApplicationPath() {
 	}
 
 	/**
-	 * Set the quarkus.http.root-path property
+	 * Set the @ApplicationPath annotation valye
 	 *
 	 * @param applicationPath as the @ApplicationPath annotation value
 	 */
 	public void setApplicationPath(String applicationPath) {
 		this.applicationPath = applicationPath;
 	}
 
+	/**
+	 * Get the Java CodeLens Context
+	 *
+	 * @return the Java CodeLens Context
+	 */
+	public JavaCodeLensContext getJavaCodeLensContext() {
+		return javaCodeLensContext;
+	}
+
+	/**
+	 * Set the quarkus.http.root-path property
+	 *
+	 * @param javaCodeLensContext
+	 */
+	public void setJavaCodeLensContext(JavaCodeLensContext javaCodeLensContext) {

```

Comment:
Remove this method and set the javaCodeLensContext in the constructor of JaxrsContext.